### DISCUSSION TITLE
Projection

### DISCUSSION THREAD

**Discussion by emaildipen** [pydata/xarray#6410] [General]
I got around 3000 GEOTIFF with CRS WGS84. I want to clip all the files at once, and project to GDA2020. How can I do it? I wrote codes to clip, but I didn't find how to project it. This is my sample code.
DB=xr.concat([xr.open_rasterio(i).sel(x=x2, y=y2, method='nearest') for i in DB_BRDF_Corrected], dim=time_var)
Where,
y2=xr.DataArray([-42.845976, -42.850294, -42.854612, -42.85893 , -42.863248, -42.867566,       -42.871884, -42.876202 ], dims='y')
x2=xr.DataArray([147.197913, 147.202231, 147.206549, 147.210867, 147.215185, 147.219503,       147.223821, 147.228139, 147.232457, 147.236775], dims='x')
DB_BRDF_Corrected=glob.glob(r'P:\DB_BRDF_Corrected**.tif'

I want to reproject and maintain the same pixel size as WGS84.

**Comment by max-sixty** [comment]
Thanks for the question — please take a look at https://github.com/pydata/xarray/discussions/5404

**Comment by keewis** [comment]
in general, `xr.open_rasterio` is deprecated in favor of [`rioxarray`](https://github.com/corteva/rioxarray), which should also have most of the features you're looking for. If you have questions on that, I'd suggest asking on their discussions board.

If that's not what you're looking for we need a bit more information (as mentioned in https://github.com/pydata/xarray/discussions/6410#discussioncomment-2432034)
